docs/pages/versions/unversioned/sdk/firebase-analytics.md

```diff
@@ -37,10 +37,28 @@ The use of Native Firebase Analytics requires that the google-services configura
 and linked into your app. Since the standard Expo Client loads projects on demand, it does not have
 the google-services configuration linked into its app-bundle.
 
-Instead, the standard Expo Client logs the analytics events to the console for debug & development
-purposes. It also generates a one time warning to let you know that analytics events are not recorded
-on the standard Expo Client. You can suppress this warning and disable the logging
-by calling `setUnavailabilityLogging(false)`.
+Instead, the standard Expo Client relies on a pure JavaScript implementation of Firebase Analytics
+to log events. This means that certain native lifecyle events are not recorded in the standard client,
+but you can still use `logEvent` to log events.
+
+In order to use Firebase Analytics on the standard Client, ensure that the firebase web-configuration
+is set in `app.json` and that `measurementId` exists in your firebase config. If `measurementId` 
+doesn't exist, then you need to enable or update Google Analytics in your Firebase project.
```

Comment:
collapse paragraphs into one line so we don't add unnecessary linebreaks to docs